code/datums/supplypacks/contraband.dm

```diff
@@ -7,7 +7,7 @@
 /datum/supply_packs/randomised/contraband
 	num_contained = 5
 	contains = list(
-			/obj/item/seeds/bloodtomatoseed,
```

Comment:
Not sure why you removed the indent here

Reply:
DM hates me